lib/arclight/custom_document.rb

```diff
@@ -45,6 +45,7 @@ def to_solr(solr_doc = {})
       Solrizer.insert_field(solr_doc, 'date_range', formatted_unitdate_for_range, :facetable)
       Solrizer.insert_field(solr_doc, 'access_subjects', access_subjects, :facetable)
       Solrizer.insert_field(solr_doc, 'all_subjects', all_subjects, :symbol)
+      Solrizer.insert_field(solr_doc, 'has_online_content', online_content?, :displayable)
```

Comment:
Anybody have any good ideas on how to reduce the complexity of this method?

I'm thinking that we could build up an array of hashes representing these fields and just iterate through that array.  Alternatively, we make this the impetus for generating a `.rubocop_todo.yml`.